vunit/modelsim_interface.py

```diff
@@ -276,7 +276,8 @@ def _create_load_function(self,  # pylint: disable=too-many-arguments,too-many-l
         else:
             coverage_file = join(output_path, "coverage.ucdb")
             self._coverage_files.add(coverage_file)
-            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" % fix_path(coverage_file)
+            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" \
+			% (os.path.basename(os.path.dirname(output_path)), fix_path(coverage_file))
```

Comment:
I agree with you that the best way is to use a test name variable, but this parameter is not passed to the SimulationInterface or ModelsimInterface class. And I don't have enough experience with python and Vunit that will allows me to do the necessary updated to add this variable to these classes.
Maybe someone else could help with adding this variable so to correct the merge error.

But I think even if the output directory changes, in any case it will be unique for a test, isn't it?